projects/angular/src/accordion/accordion-panel.ts

```diff
@@ -111,6 +111,14 @@ export class ClrAccordionPanel implements OnInit, OnChanges {
     return `clr-accordion-header-${id}`;
   }
 
+  stepCompleteText(panelNumber: number) {
```

Comment:
```suggestion
  protected stepCompleteText(panelNumber: number) {
```

You can mark these methods `protected` so that they don't show up in public API.